src/web/transforms/images.ts

```diff
@@ -58,41 +80,19 @@ export async function transformImages(mdast: Root, state: TransformState) {
         // Assume non-oxa, non-url paths are local images relative to the config.section.path
         const fullPath = path.join(state.folder, image.url);
         if (!fs.existsSync(fullPath)) {
-          session.log.debug(`Cannot find image: ${fullPath}`);
+          session.log.error(`Cannot find image: ${fullPath}`);
```

Comment:
No errors on external / oxa images, though? Makes some sense, since the user has more control over local images.

Reply:
Will cover the other ones as well!